### ISSUE TITLE
JAWS doesn't allow you to delete parts of expressions in the math editor

### ISSUE THREAD

**Issue by tayarndt** [arnog/mathlive#507]
## Prerequisites

        [Did you check the [issue tracker](https://github.com/arnog/mathlive/issues) to see if this issue has already been reported? If it has, you can +1 it to indicate your interest and be notified when it gets resolved.]

Yes

## Description
After you are done typing an expression with JAWS in the math live math  field,you are unable to delete parts of the expression when pressing backspace.

### Actual behavior
The actual behavior is after pressing backspace, JAWS doesn't indicate that the term or terms have been removed. Instead, JAWS says nothing.

### Expected Behavior
The expected behavior is when the user selects the term, and the user presses backspace that JAWS would say deleted to indicate the term is gone.

###Is this a regression: did it use to work in a previous version?
Yes, this did. At one point, i was able to get this to work. Unfortunately, i don't have a version in which it stopped working.

## Steps to Reproduce
1. turn on JAWS.
2. next, find the math editor on the website.
3. type an expression.
4. try to delete term or terms by selecting and pressing backspace.

### Environment
Windows 10 version 2004.
Google chrome latest version
https://mathlive.io/
I have an audio demo however github isn't allowing me to upload.

**Comment by NSoiffer** [comment]
I just tried this with both NVDA and JAWS (Version 2019.1909.28 ILM) and it
announced the delete in both cases when using mathlive.io (mathlive.js
 v0.53.2-0-gdba9a49-dirty).

**Comment by tayarndt** [comment]
Okay, i was able to reproduce with JAWS2020. I don’t have 2019,but have the latest 2020.

**Comment by NSoiffer** [comment]
When did you upgrade JAWS? You said it use to say "delete " for you. Maybe
the JAWS upgrade is what changed... and JAWS 2020 has a bug.

mathlive make use of aria-live regions. Hard to imagine that broke in JAWS,
but it does work with the older  version and with NVDA.

**Comment by tayarndt** [comment]
Here is the thing, my screen reader of choice is NVDA. I am helping with the application mathshare and had to do some JAWS testing to see if the issue was a mathshare issue, a JAWS issue, or a mathlive issue. I am going to see if I can download JAWS 2019 and see if those live regions are broken and if so I will have to make a call to freedom scientific.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
